node_modules/oae-core/filepreview/js/filepreview.js

```diff
@@ -80,6 +80,20 @@ define(['require', 'jquery', 'oae.core'], function(require, $, oae) {
         };
 
 
+        /////////////////////
+        //  VIDEO PREVIEW  //
+        /////////////////////
+
+        /**
+         * Render a preview of a video file
+         */
+        var renderVideoPreview = function() {
```

Comment:
When ZenCoder is processing the video, we see the `Processing` message in the UI. However, when processing has finished, the UI should automatically be showing the preview following a push notification. It's doing this for other previews, but doesn't appear to be doing this for videos.